build.sbt

```diff
@@ -1,5 +1,7 @@
 import java.io.File.pathSeparator
 
+import sbt.Keys.libraryDependencies
+
```

Comment:
This import is not necessary. `sbt.Keys._` is already imported implicitly in `build.sbt`.